packages/manager/apps/pci-block-storage/src/pages/list/guidesHeader.constants.ts

```diff
@@ -0,0 +1,104 @@
+import { Guide } from '@ovh-ux/manager-react-components/dist/types/src/components/guides-header/interface';
```

Comment:
I think this shouldn't be imported like this since it can be removed at anytime

You can find a typing inside the constant file

Reply:
done